scylla-server/src/transformers/run_transformer.rs

```diff
@@ -19,51 +14,13 @@ pub struct PublicRun {
     pub time_ms: i64,
 }
 
-impl From<&public_run::Data> for PublicRun {
-    fn from(value: &public_run::Data) -> Self {
-        PublicRun {
-            id: value.id,
-            location_name: value.location_name.clone().unwrap_or_default(),
-            driver_name: value.driver_name.clone().unwrap_or_default(),
-            system_name: value.system_name.clone().unwrap_or_default(),
-            time_ms: value.time.timestamp_millis(),
-        }
-    }
-}
-
-// why are these three needed? basically the nested relations via select do not "share" nested relations elsewhere.
-// ultimately this means structs with identical fields have non identical types, and as they are macro generated they cannot be derived together
-impl From<&public_driver::runs::Data> for PublicRun {
-    fn from(value: &public_driver::runs::Data) -> Self {
-        PublicRun {
-            id: value.id,
-            location_name: value.location_name.clone().unwrap_or_default(),
-            driver_name: value.driver_name.clone().unwrap_or_default(),
-            system_name: value.system_name.clone().unwrap_or_default(),
-            time_ms: value.time.timestamp_millis(),
-        }
-    }
-}
-
-impl From<&public_location::runs::Data> for PublicRun {
-    fn from(value: &public_location::runs::Data) -> Self {
-        PublicRun {
-            id: value.id,
-            location_name: value.location_name.clone().unwrap_or_default(),
-            driver_name: value.driver_name.clone().unwrap_or_default(),
-            system_name: value.system_name.clone().unwrap_or_default(),
-            time_ms: value.time.timestamp_millis(),
-        }
-    }
-}
-
-impl From<&public_system::runs::Data> for PublicRun {
-    fn from(value: &public_system::runs::Data) -> Self {
+impl From<crate::models::Run> for PublicRun {
+    fn from(value: crate::models::Run) -> Self {
         PublicRun {
             id: value.id,
-            location_name: value.location_name.clone().unwrap_or_default(),
-            driver_name: value.driver_name.clone().unwrap_or_default(),
-            system_name: value.system_name.clone().unwrap_or_default(),
+            location_name: value.locationName.unwrap_or_default(),
+            driver_name: value.driverName.clone().unwrap_or_default(),
+            system_name: value.driverName.unwrap_or_default(), // TODO no system name
```

Comment:
What's going on here?

Reply:
we need system name or else client wont like it.  @bracyw can u fix client?